apps/api-extractor/src/DebugRun.ts

```diff
@@ -2,30 +2,61 @@
 // See LICENSE in the project root for license information.
 
 // NOTE: THIS SOURCE FILE IS FOR DEBUGGING PURPOSES ONLY.
-//       IT IS INVOKED BY THE "Run.cmd" AND "Debug.cmd" BATCH FILES.
+//       IT IS INVOKED BY THE 'Run.cmd' AND 'Debug.cmd' BATCH FILES.
 
 import { Extractor } from './extractor/Extractor';
+import * as path from 'path';
 
 const extractor: Extractor = new Extractor(
   {
     compiler: {
       configType: 'tsconfig',
-      rootFolder: '.'
+      overrideTsconfig: {
+        'compilerOptions': {
+          'target': 'es6',
+          'forceConsistentCasingInFileNames': true,
+          'module': 'commonjs',
+          'declaration': true,
+          'sourceMap': true,
+          'experimentalDecorators': true,
+          'types': [
+            'node'
+          ],
+          'lib': [
+            'es5',
+            'scripthost',
+            'es2015.collection',
+            'es2015.promise',
+            'es2015.iterable',
+            'dom'
+          ],
+          'strictNullChecks': true
+        },
+        'include': [ 'lib/**/*.d.ts' ]
```

Comment:
>'include': [ 'lib/**/*.d.ts' ] [](start = 8, length = 30)

Go figure out how this is going to work with GCB.

Reply:
I've got a working prototype of this, but it needs to go in a separate PR branch.  It may also require a major version bump for API Extractor.

---
In reply to: [152421152](https://github.com/Microsoft/web-build-tools/pull/448#discussion_r152421152) [](ancestors = 152421152)